circle.yml

```diff
@@ -1,8 +0,0 @@
-machine:
```

Comment:
Doesn't seem like Circle should be excluded. Are we not linting each PR or running tests? It looks like there's one test file in this PR and there are affordances for testing in the webpack config

Reply:
k, i'll re-add Circle. I'll re-add testing when we actually have tests. The one test that Max wrote is completely outdated.